samples/client-sdk/getting-started-daily-js/index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Daily Call Example</title>
+    <link href="style.css" rel="stylesheet" />
+  </head>
+  <body>
+    <div>
+      <label for="room-url">Room URL:</label>
+      <input
+        type="text"
+        id="room-url"
+        size="50"
+        placeholder="https://yourcompany.daily.co/hello"
+      />
+    </div>
+    <div>
+      <label for="room-url">Token:</label>
+      <input
+        type="text"
+        id="join-token"
+        size="50"
+        placeholder="Enter Join Token (optional)"
```

Comment:
Sorry I didn't notice this before, but I wonder if we should actually make this say "Enter Daily meeting token" (and link the text to our meeting token docs or guide). To remain consistent and clear with the terminology.

Reply:
Good idea!